.gitignore

```diff
@@ -113,3 +113,6 @@ webnews/
 webroster/
 ybook2003/
 yearbook/
+
+# Ignore Config File from git
+data/config.json
```

Comment:
Could you have the file end with a newline? For reference on why this is important please visit: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline